### DISCUSSION TITLE
Possibility to add payload field

### DISCUSSION THREAD

**Discussion by Perwol** [Koenkk/zigbee2mqtt#6842] [General]
Hi!
Is there a way to add some custom field to payload got as Json MQTT response? I would like to add the same field to every response, for example:

`{ "state": "ON", "custom_data": "some_data" }`

It is possible now? There is no documentation about this.

**Comment by Koenkk** [comment]
It's not supported (and I don't see a usecase for it) but in case the data is static you can add it into `state.json` and z2m will publish it.
